### ISSUE TITLE
User's choice for "reply to identity" is not honored

### ISSUE THREAD

**Issue by migux** [thunderbird-conversations/thunderbird-conversations#1903]
* **Thunderbird version:** 102.3.0
* **Thunderbird Conversation version:** 4.0.22
* **Operating System:** Win 10

**What did you do?**

1. Create an additional identity in TB account settings and set up catch all function "Reply from this identity when delivery headers match"
2. Reply to an email (received for identity X in the account for identity A)

**What did you expect to happen?**
TB should select identity X as sender

**What actually happened?**
TB selects the default identity from this account
_It work as expected when I open the message in "classic view" (klassische Ansicht) and then hit reply_ 

**Reference for catch All function in TB**
https://bugzilla.mozilla.org/show_bug.cgi?id=1518025

**Comment by Standard8** [comment]
Thanks for the submission. Unfortunately at the moment Thunderbird doesn't provide an easy way for extensions to get the required information about the identity. I've filed [a bug about that](https://bugzilla.mozilla.org/show_bug.cgi?id=1793336). Once that has been implemented, we can look at respecting that setting.

**Comment by jobisoft** [comment]
@migux : Is forwarding in the standard UI (without conversations) also honouring this setting, or is it limited to replies?

**Comment by migux** [comment]
@jobisoft: Yes forwarding also honours this setting. When forwarding, TB selects the defined catch all identity as sender.
